main.js

```diff
@@ -212,6 +212,14 @@ function selectMove(event) {
 }
 
 function newChallenger() {
+  if (!currentGame.currentZodiac.length) {
+    hide(computerBox)
+    hide(battleText)
+    playerBox.innerHTML = ""
```

Comment:
Since innerHTML = "" is being used so much, maybe it can be incorporated to a function like "resetInner"?